dev-support/.scalafmt.conf

```diff
@@ -16,18 +16,39 @@
 #
 
 # Template based off apache spark: https://github.com/apache/spark/blob/master/dev/.scalafmt.conf
+# Align settings
 align = none
 align.openParenDefnSite = false
 align.openParenCallSite = false
 align.tokens = []
+
+# Rewrites
+rewrite.rules = [Imports]
+
+# Imports
+rewrite.imports.sort = scalastyle
+rewrite.imports.groups = [
```

Comment:
This order is based on what we have for java. See https://github.com/apache/hbase-connectors/blob/master/dev-support/eclipse.importorder